### PR TITLE
Adding correct files, to fix some common problems found in some scrip…

### PR DESCRIPTION
**Pull request by allancm**
Adding correct files, to fix some common problems found in some scripts for some customers to avoid unnecessary parsing errors.

Support for fload snippet generation in the extract snippets (and support for a different extension snippet from the tag found in the script).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ In those scenarios you can use these helpers scripts. You can run them like this
 python extract_snippets.py -h
 usage: extract_snippets.py [-h] --inputdir INPUTDIR --outdir OUTDIR --verbose
 
-BTEQ/MLOAD embeded shell script extractor for SnowConvert
+BTEQ/MLOAD/FLOAD embeded shell script extractor for SnowConvert
```
Comment (orellabac) [inline README.md:60]: fix type: embedded

File: correct_files.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
+##########################################################################################################
+#
+#  This is a helper utility to process some input files for the conversion tool named SnowConvert
+#  of mobilize.net
```
Comment (orellabac) [inline correct_files.py:4]: Mobilize.Net

File: correct_files.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+#
+#  More info: https://www.mobilize.net/products/database-migrations/teradata-to-snowflake
```
Comment (orellabac) [inline correct_files.py:6]: use the more generic link: https://www.mobilize.net/products/database-migrations/snowconvert

File: correct_files.py [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+#
+#   *4) End double quote at the end of last line of the file
+#       Input:
+#         LOGOFF"
+#       Output:
+#         LOGOFF
+#         --"
+#
+# Changes Log
+# Version 1.0.0
```
Comment (orellabac) [inline correct_files.py:48]: These comments should be part of the changes log. What I mean is Add a Changes log Header and include these descripcios as para of the improvements

File: extract_snippets.md [hunk 5]
```diff
@@ -37,9 +37,9 @@ The first encoding that works will be used and their generated files will also u
 The summary of the output is a csv containing the followin columns:
```
Comment (orellabac) [inline extract_snippets.md:37]: following

File: extract_snippets.md [hunk 6]
```diff
@@ -37,9 +37,9 @@ The first encoding that works will be used and their generated files will also u
...
 * ENCODING: the encoding that the file was read
 * FILE: the full file path name of the file
-* HAS_EMBEDDED: True if the file has embedded blocks of bteq or mload, False otherwise
+* HAS_EMBEDDED: True if the file has embedded blocks of bteq, fload or mload, False otherwise
```
Comment (orellabac) [inline extract_snippets.md:40]: what about tpt?
or tump?

Comment (allancm) [reply extract_snippets.md:40]: Still unsupported, when we found how the tag is used in a real example we will added (like with this case of fastload)

File: correct_files.py (hunk 2 above)
Comment (allancm) [reply correct_files.py]: fixed thanks

File: correct_files.py (hunk 4 above)
Comment (allancm) [reply correct_files.py]: fixed thanks

File: extract_snippets.md (hunk 5 above)
Comment (allancm) [reply extract_snippets.md]: fixed thanks

File: correct_files.py (hunk 3 above)
Comment (allancm) [reply correct_files.py]: fixed thanks

**Pull request merged**